packages/fragments.rb

```diff
@@ -36,6 +36,7 @@ class Fragments < Package
   depends_on 'openssl' # R
   depends_on 'pango' # R
   depends_on 'rust' => :build
+  depends_on 'transmission' # L
```

Comment:
Do we need new binaries?

Reply:
No. The software just doesn't throw an error when transmission isn't installed. It's an upstream bug.